modules/katamari/src/main/ts/ephox/katamari/api/Maybes.ts

```diff
@@ -0,0 +1,327 @@
+import { Optional } from './Optional';
```

Comment:
This appears to be missing a few implementations from `Optionals`. Namely `cat`, `mapFrom`, `sequence` `someIf` and `traverse`. Those will be needed if we are going to replace Optionals.

- [ ] Add missing implementations from `Optionals`

Reply:
I reckon we Jira these up for a later task, so we can get this PR merged.

Reply:
Jira has been made